src/pages/tasks/NewTaskDetailsPage.js

```diff
@@ -20,15 +20,33 @@ const propTypes = {
     /** Beta features list */
     betas: PropTypes.arrayOf(PropTypes.string),
 
+    /** Task title and description data */
+    task: PropTypes.shape({
+        title: PropTypes.string,
+        description: PropTypes.string,
+    }),
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     betas: [],
+    task: {},
 };
 
 const NewTaskPage = (props) => {
     const inputRef = useRef();
+    const [taskTitle, setTaskTitle] = useState(props.task.title);
+    const [taskDescription, setTaskDescription] = useState(props.task.description);
+
+    useEffect(() => {
+        if (taskTitle !== props.task.title) {
```

Comment:
Do we need to compare?

Reply:
We don't need this since the `useEffect` call will run only once when the component mounts.